package.json

```
@@ -71,9 +71,9 @@
   "devDependencies": {
     "@cypress/skip-test": "2.6.0",
     "@testing-library/cypress": "8.0.1",
-    "cypress": "9.5.1",
+    "cypress": "11",
```

For consistency let's add the full semver.  From looking in the `node_modules` folder, it looks like we're already on `11.2.0`.  

```suggestion
    "cypress": "11.2.0",
```

---

Has been updated.